controllers/gamesController.js

```diff
@@ -1,11 +1,23 @@
-const { selectCategories } = require("../models/gameModels");
+const { selectCategories, selectReviews } = require("../models/gameModels");
 
-exports.getCategories = (req, res, next) => {
+exports.getCategories = (request, response, next) => {
   selectCategories()
     .then((category) => {
-      res.status(200).send(category);
+      response.status(200).send(category);
     })
-    .catch((err) => {
-      next(err);
+    .catch((error) => {
+      next(error);
+    });
+};
+
+exports.getReviews = (request, response, next) => {
+  const { sort_by } = request.query;
+
+  selectReviews(sort_by)
+    .then((review) => {
+      response.status(200).send(review);
```

Comment:
Minor change - if we are sending more than one review, it would make sense to send it on the key of review(s) plural as opposed to review.